mlrun/projects/project.py

```diff
@@ -1967,6 +1968,7 @@ def run(
                           for using the pre-defined workflow's schedule, set `schedule=True`
         :param timeout:   timeout in seconds to wait for pipeline completion (used when watch=True)
         :param overwrite: replacing the schedule of the same workflow (under the same name) if exists with the new one.
+        :param source:    remote source to overwrite the actual `project.spec.source` (used when engine is remote).
```

Comment:
why we want to override the `project.spec.source` on run method?

Reply:
you right, its not exactly overwrite... will rephrase, see 4. in description